compliance/compliance-server/src/db.rs

```diff
@@ -25,12 +28,20 @@ use crate::{
 pub struct ComplianceEntry {
     pub address: String,
     pub is_compliant: bool,
+    pub risk_level: String,
     pub reason: String,
     pub created_at: SystemTime,
     pub expires_at: SystemTime,
 }
 
 impl ComplianceEntry {
+    /// Create a new entry from a risk assessment
+    pub fn new(address: String, is_compliant: bool, risk_level: String, reason: String) -> Self {
```

Comment:
nit: why not have this accept a `RiskAssessmentResponse` directly? will read more nicely

Reply:
separation of concerns; we should be able to create a `ComplianceEntry` without needing to construct a chainalysis response. I see no reason to couple them.